GsaGHTests/1_BaseParameters/1_Properties/GsaBool6Test.cs

```diff
@@ -83,5 +88,67 @@ public void DuplicateTest(bool x, bool y, bool z, bool xx, bool yy, bool zz) {
       Assert.Equal(yy, original.Yy);
       Assert.Equal(zz, original.Zz);
     }
+
+    [Fact]
+    public void ReleaseParameterInfoIsCorrect() {
```

Comment:
split to include one assert per test

Reply:
@psarras @DominikaLos   I think when checking one output against one input then one assert make sense. Here single creation of object return think which we want to test. Checking name, description, nickname in separate test will make  it lengthy and multiple call of same thing.